librz/cons/dietline.c

```diff
@@ -1680,7 +1680,6 @@ RZ_API const char *rz_line_readline_cb(RzLineReadCallback cb, void *user) {
 			__move_cursor_right();
 			break;
 		case 12: // ^L -- right
-			__move_cursor_right();
```

Comment:
Why is that change? According to the comment it should move right, no?

Reply:
Indeed, I forgot to update the comment.  I'll fix that.

Or do you disagree about Ctrl-L not being supposed to move the cursor?